src/views/clusteroverview/SettingsTab/ClusterTab/components/ResourceManagementSection/components/AutoComputeCPULimits/AutoComputeCPULimits.tsx

```diff
@@ -34,50 +37,48 @@ const AutoComputeCPULimits: FC = () => {
 
   useEffect(() => {
     if (!autoComputeCPUPreviewEnabled) toggleCPULimits(false);
-  }, [autoComputeCPUPreviewEnabled]);
+  }, [autoComputeCPUPreviewEnabled, toggleCPULimits]);
```

Comment:
Did you check to make sure this didn't cause unnecessary renders? I think that's why I left that off, but I'm not 100% sure. I tried to test it, but my system is hosed right now.